static/playground/index.html

```diff
@@ -0,0 +1,15 @@
+<html>
+
+<head>
+  <style>
+    body {
+      margin: 0;
+    }
+  </style>
+</head>
+
+<body>
+  <iframe src="https://play.fga.dev/sandbox/?fga_api_host={{.HTTPServerURL}}&fga_api_scheme=http" frameborder="0" marginheight="0" marginwidth="0" width="100%" height="100%" scrolling="auto"></iframe>
+</body>
+
+</html>
```

Comment:
Newline?

Reply:
NBD, but can remove it

Reply:
I meant should we add one? It gets rid of that "no newline" mark which always bothers me (but I think I am the only one).